tests/nodeos_eos_evm_test.py

```diff
@@ -911,22 +923,30 @@ def get_block(num):
     signed_trx = w3.eth.account.sign_transaction(dict(
         nonce=nonce,
         gas=100000,       #100k Gas
-        gasPrice=900000000000,
+        maxFeePerGas = 900000000000,
+        maxPriorityFeePerGas = 900000000000,
+        #gasPrice=900000000000,
         to=Web3.to_checksum_address(toAdd),
         value=int(amount*10000*szabo*100), # .0001 EOS is 100 szabos
         data=b'',
         chainId=evmChainId
     ), evmSendKey)
+    Print("EVM transaction hash is: %s" % (Web3.to_hex(signed_trx.hash)))
     actData = {"miner":minerAcc.name, "rlptx":Web3.to_hex(signed_trx.rawTransaction)[2:]}
     trans = prodNode.pushMessage(evmAcc.name, "pushtx", json.dumps(actData), '-p {0}'.format(minerAcc.name), silentErrors=False)
     prodNode.waitForTransBlockIfNeeded(trans[1], True)
     row4=prodNode.getTableRow(evmAcc.name, evmAcc.name, "account", 4) # 4th balance of this integration test
-    Utils.Print("\taccount row4: ", row4)
+    Utils.Print("\tverify balance from evm-rpc, account row4: ", row4)
     bal2 = w3.eth.get_balance(Web3.to_checksum_address("0x9E126C57330FA71556628e0aabd6B6B6783d99fA"))
 
     # balance different = 1.0 EOS (val) + 900(Gwei) (21000(base gas) + 36782 (gas for non-exist account) )
     assert(bal1 == bal2 + 1000000000000000000 + 900000000000 * (21000 + 36782))
 
+    Utils.Print("try to get transaction %s from evm-rpc" % (Web3.to_hex(signed_trx.hash)))
+    evm_tx = w3.eth.get_transaction(signed_trx.hash)
```

Comment:
what happens here if transaction does not exists?
Maybe we can assert that : signed_trx.hash == evm_tx hash

We should also call eth_getTransactionReceipt and do something similar

Reply:
Also the PR should start with `[1.0] ...` and target `release/1.0`